### PR TITLE
Remove constant usages from TimberTheme

### PR DESCRIPTION
**Pull request by xavivars**
According to WordPress documentation, path-related constants shouldn't be used by plugin/themes, and it seems the issue was because Timber was assuming the themes folder would always be inside WP_CONTENT_DIR.

By using methods instead of the constants, `{{theme.path}}` returns the right value.

This should fix `PauGNU`'s issue in #619

### CODE REVIEW

**Comment by jarednova** [comment]
@xavivars thanks for the find! As I keep working on this I'm finding all kids of ways to configure WP setups (and all kinds of ways to Timber to mix-up paths in the process). 

Could you update this PR with tabs instead of spaces for the indentation? From here out I'm following WordPress's [PHP Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/) to play (mostly) nicely with the rest of the code.

**Comment by xavivars** [comment]
Hi @jarednova, I've updated the pull request. When you have time, take a look at it.

Thanks!

**Comment by xavivars** [comment]
Hi, I've just found another (related) bug: when changing the UPLOADS constant to be outside of WP-CONTENT, `TimberURLHelper` was detecting the image as an external image. With this patch this is no longer happening.

I've tried to run the unit tests, to see if there's anything broken, and also to add tests to cover this case, but I couldn't run it: it seems I'm missing something to set up your tests environment, but I don't see documented how to set it up.

File: lib/timber-url-helper.php [hunk 1]
```diff
@@ -225,14 +225,25 @@ public static function is_absolute( $path ) {
...
-		// 2. or redefine upload directory
-		$is_external = TimberURLHelper::is_absolute( $url ) && !strstr( $url, content_url() );
-		return $is_external;
-	}
+        $is_external = TimberURLHelper::is_absolute( $url ) && ! TimberURLHelper::is_internal_content( $url );
+                
+        return $is_external;
+    }
+
+    private static function is_internal_content($url) {
```
Comment (prisis) [inline lib/timber-url-helper.php]: need more spaces

Comment (xavivars) [inline lib/timber-url-helper.php]: Thanks, just fixed it.

**Comment by xavivars** [comment]
Just added unit tests to it.

**Comment by jarednova** [comment]
@xavivars -- thanks for the PR and the tests! It's now merged and will release tomorrow to wp.org

**Pull request merged**